backend/apps/notifications/channels/push_channel.py

```
@@ -0,0 +1,62 @@
+                    },
+                    data=push_payload,
+                    vapid_private_key=vapid_private_key,
+                    vapid_claims={"sub": f"mailto:{vapid_admin_email}"},
+                )
+                delivered_any = True
+            except WebPushException as ex:
+                if ex.response and ex.response.status_code in [404, 410]:
+                    sub.delete()
+                else:
+                    logger.warning("WebPush failed for sub %s: %s", sub.id, ex)
+            except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)